helmfile/charts/elasticsearch-slm/scripts/slm-retention.bash

```diff
@@ -75,12 +76,14 @@ function remove_old_snapshots {
     local idx=0
     local snapshots
     local snapshot_count
+    local snapshots_to_delete=""
 
     snapshots=$(get_snapshots)
     snapshot_count=$(echo "${snapshots}" | jq length)
+    echo "Number of snapshots $snapshot_count"
```

Comment:
nit: I would move this to `remove_excess_snapshots`. It is nice for debugging, but doesn't make sense when checking for age.
```suggestion
```